CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ _unreleased_
 
 - Associate `.doccls` file extension
 - Engineering: Switch from npm to pnpm
+- Limit double quotes `"` with one line
```

Comment:
Please move this line to the top as it is more significant than the other changes.

And I think a text like `Limit string highlighting to a single line` is slightly more obvious